include/ccf/endpoint_registry.h

```diff
@@ -42,6 +42,51 @@ namespace ccf::endpoints
   void default_locally_committed_func(
     CommandEndpointContext& ctx, const TxID& tx_id);
 
+  template <typename T>
```

Comment:
That just seems to have moved, but is otherwise unchanged?

Reply:
With hindsight, this is a breaking change. We should make this static in the class (I think static is all it needed?), and/or a shim so that the old calls still work.